pyproject.toml

```diff
@@ -1,21 +1,18 @@
-[tool.poetry]
+[project]
```

Comment:
Please, revert pyproject.toml changes to this new PEP thing, it does not work with dependabot. And we kinda need it :)